XamlControlsGallery/ControlPages/AnimatedVisualPlayerPage.xaml

```diff
@@ -70,27 +71,30 @@
                                 <SymbolIcon x:Name="PauseIcon" Symbol="Pause" Visibility="Collapsed"/>
                             </StackPanel>
                         </Button>
-                        
-                        <!--  Stop/Reset -->
+
+                        <!-- Reset -->
                         <Button x:Name="StopButton"
                             Grid.Column="1"
                             Margin="10"
                             HorizontalAlignment="Stretch"
                             Click="StopButton_Click"
                             ToolTipService.ToolTip="Stop"
                             AutomationProperties.Name="Stop">
-                            <SymbolIcon Symbol="Previous" />
+                            <SymbolIcon Symbol="Stop" />
                         </Button>
-                        
-                        <!--  Reverse  -->
-                        <Button x:Name="ReverseButton"
+
+                        <!--  Normal/Reverse play  -->
```

Comment:
Instead of "normal" what about "forward"?

Reply:
Good point, no idea why I went for normal here, but forwards everywhere else. Fixed now.